pkg/apis/core/helper/helpers.go

```diff
@@ -550,6 +550,34 @@ func ConvertSeedExternal(obj runtime.Object) (*gardencorev1beta1.Seed, error) {
 	return result, nil
 }
 
+// ConvertSeedTemplate converts the given external SeedTemplate version to an internal version.
+func ConvertSeedTemplate(obj *gardencorev1beta1.SeedTemplate) (*core.SeedTemplate, error) {
+	seed, err := ConvertSeed(&gardencorev1beta1.Seed{
+		Spec: obj.Spec,
+	})
+	if err != nil {
+		return nil, errors.New("could not convert SeedTemplate to internal version")
+	}
+
+	return &core.SeedTemplate{
+		Spec: seed.Spec,
+	}, nil
+}
+
+// ConvertSeedTemplateExternal converts the given external SeedTemplate version to an external version.
```

Comment:
```suggestion
// ConvertSeedTemplateExternal converts the given internal SeedTemplate version to an external version.
```

I guess you mean internal here